File: C#/1ere année/ClassesValiers/ClassesValiers/Categorie.cs
using System;


namespace ClassesValiers
{
    public class Categorie
    {
        /// <summary>
        /// propriétés privées
        /// </summary>
        private string numCategorie;
        private string nomCategorie;

        /// <summary>
        /// constructeur
        /// </summary>
        public Categorie(string num, string nom)
        {
            this.numCategorie = num;
            this.nomCategorie = nom;

        }




        /// <summary>
        /// retourne le nom de la catégorie
        /// </summary>
        public string NomCategorie
        {
            get => nomCategorie;
        }
    }
}

File: JAVA/casvaliers/classescasvaliers/Categorie.java
package classescasvaliers;

public class Categorie {

    /**
     * propriétés privées
     */
    private String numCategorie;
    private String nomCategorie;

    /**
     * Constructeur de la classe
     *
     * @param n numéro de la catégorie
     * @param nom nom de la catégorie
     */
    public Categorie(String n, String nom) {
        numCategorie = n;
        nomCategorie = nom;
    }

    /**
     * accesseur pour le nom de la catégorie
     *
     * @return le nom de la catégorie
     */
    public String getNomCategorie() {

        return nomCategorie;

    }

}
